templates/santa/finish.html.twig

```diff
@@ -4,24 +4,67 @@
     <div class="content result-block">
         {% if secretSanta.isDone %}
             <h2>Well done! All messages were sent</h2>
+
+            <p class="summary">
+                You have received a private message explaining how to retrieve the santa repartition. You should probably don't look at it, to avoid breaking the Secret Santa spirit.
+            </p>
         {% else %}
             <h2>A technical error occurred when sending messages to users</h2>
 
             <div class="result-error">
                 <p>
-                    "{{ secretSanta.errors|last }}"
+                    Sometimes, even Santa run into problems when doing its job. But keep reading, everything is not lost!
+                </p>
+
+                <p>
+                    You will find below some <strong>important information</strong> and <strong>a way to safely retry</strong> the send.
                 </p>
 
+                <h3>The technical error</h3>
+
+                <p>
+                    Here is the evil error that caused this fail:
+                </p>
+
+                <div class="error-code">
+                    {% for error in secretSanta.getUniqueErrors() %}
+                        <span>{% if loop.length > 1 %}- {% endif %}{{ error }}</span><br />
```

Comment:
Why don't you use ul+li ?

Reply:
Because in most cases, there will be only one error to display, so I didn't want to suggest there could be more than one with a list bullet :stuck_out_tongue: